doc/radosgw/config-ref.rst

```diff
@@ -1115,6 +1115,39 @@ Keystone Settings
 :Type: Boolean
 :Default: ``true``
 
+
+``rgw enable lc threads``
```

Comment:
could you add a section header for these so they're not confused with the `Keystone Settings` section above?